core/email.py

```diff
@@ -141,7 +145,7 @@ def sendEMail(*,
               headers: Dict[str, str] = None,
```

Comment:
`dests`, `cc` and `bcc` now can be a `Callable[[], List]]` as well?

Reply:
Yes, the same known and accepted side effect.